products/samsungmobile.md

```diff
@@ -122,7 +347,7 @@ releases:
     releaseDate: 2021-04-23
 
 -   releaseCycle: "Galaxy F12"
-    support: true
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
```

Comment:
Upgradable to Android 13 according to https://www.gsmarena.com/samsung_galaxy_f12-10825.php. Should be marked supported.